AGS02MA.cpp

```diff
@@ -205,17 +205,34 @@ uint32_t AGS02MA::readUGM3()
 }
 
 
-bool AGS02MA::zeroCalibration()
+
+bool AGS02MA::manualZeroCalibration(uint16_t value)
 {
   _buffer[0] = 0x00;
   _buffer[1] = 0x0C;
-  _buffer[2] = 0xFF;
-  _buffer[3] = 0xF3;
-  _buffer[4] = 0xFC;
+  _buffer[2] = (uint8_t) (value >> 8);
+  _buffer[3] = (uint8_t) value;
```

Comment:
(uint8_t) (value & 0x00FF)  is more explicit

Reply:
I'm not sure if any/all of these explicit casts / masks would get compiler optimized away or cost additional cycles.
But I think explicit is better too, so using your suggested mask.